embark/entrypoint.sh

```diff
@@ -1,23 +1,65 @@
 #!/bin/bash
 
-export DJANGO_SETTINGS_MODULE=embark.settings.docker # TODO
+export DJANGO_SETTINGS_MODULE=embark.settings.docker
+export EMBARK_DEBUG=True
+export HTTP_PORT=80
+export HTTPS_PORT=443
+export BIND_IP='0.0.0.0'
+export FILE_SIZE=2000000000
 
-if ! [[ -d logs ]]; then
-  mkdir logs
+cd "$(dirname "$0")" || exit 1
+
+# check emba
+echo -e "$BLUE""$BOLD""checking EMBA""$NC"
+# TODO pipe
+
+# start venv (ignore source in script)
+# shellcheck disable=SC1091
+source ./.venv/bin/activate || exit 1
+
+# start the supervisor
+systemctl enable embark.service
+systemctl start embark.service
+
+# !DIRECTORY-CHANGE!
+cd /var/www/embark/ || exit 1
+
+# logs
+if ! [[ -d /var/www/logs ]]; then
+  mkdir /var/www/logs
 fi
 
-echo -e "[*] Setup logging of redis database - log to ./logs/redis_db.log"
-docker container logs embark_redis_1 -f > ./logs/redis_db.log &
-echo -e "[*] Setup logging of mysql database - log to ./logs/mysql_db.log"
-docker container logs embark_auth-db_1 -f > ./logs/mysql_db.log &
-echo -e "[*] Starting migrations - log to ./logs/migration.log"
-python3 manage.py makemigrations users uploader | tee -a ./logs/migration.log
-python3 manage.py migrate | tee -a ./logs/migration.log
-echo -e "[*] Starting runapscheduler"
-python3 manage.py runapscheduler --test | tee -a ./logs/migration.log &
-echo -e "[*] Starting uwsgi - log to ./logs/uwsgi.log"
-uwsgi --wsgi-file /app/embark/embark/wsgi.py --http :80 --processes 2 --threads 10 --logto ./logs/uwsgi.log &
-echo -e "[*] Starting daphne - log to ./logs/daphne.log"
-# shellcheck disable=2094
-#TODO the bind address has to be set at runtime or dns-res
-daphne -v 3 --access-log ./logs/daphne.log embark.asgi:application -p 8001 -b '0.0.0.0' &> ./logs/daphne.log
+# db_init
+echo -e "\n[""$BLUE JOB""$NC""] Starting migrations - log to embark/logs/migration.log"
+python3.10 manage.py makemigrations users uploader dashboard reporter | tee -a /var/www/logs/migration.log
```

Comment:
could we define this as variable in the beginning. Then we can see at beginning of the script which Python version is used